install_swiftformat.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Installs the SwiftFormat package.
```

Comment:
this file installs it for Travis purposes, yeah? I.e. this is a CI script and not something expected to be run locally?

Reply:
There's nothing stopping folks from running it locally to install the package too, but yeah it's primary purpose is for Travis

Reply:
I ask because it uses brew (which implies Travis uses macos) which seems atypical for CI setups. So might be worth calling out in the comment. Just a thought :)

Reply:
Oh gotcha- yeah this repo's `.travis.yml` configures it to use macOS, mostly since the majority of swift development (I assume) happens on macOS